charts/app/values.yaml

```diff
@@ -27,6 +27,11 @@ appealsServiceApi:
   config:
     port: 3000
     logLevel: trace
+    notify:
+      baseUrl: https://api.notifications.service.gov.uk
+      secretName: ""
+      serviceId: dummy-service-id-for-notify
+      templateId: 15ed37a9-506c-4845-88ea-95502282a863
```

Comment:
@frank-dodion is this the correct value for dev/preprod/prod?

Reply:
Oh, so the `serviceId` is also a sensible default that we can override. I'll make that change. Thanks m'colleague

Reply:
Yes.